### ISSUE TITLE
Error: spawn npm ENOENT

### ISSUE THREAD

**Issue by alexisab** [jadjoubran/webdash-npm-scripts#12]
I got this error when I try to launch a script 
```
λ npx webdash serve
F:\Bibu\Perso\myproj\api\node_modules\webdash\scripts\index.js
Webdash running on http://localhost:3456!
events.js:137
      throw er; // Unhandled 'error' event
      ^

Error: spawn npm ENOENT
    at _errnoException (util.js:1003:13)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:201:19)
    at onErrorNT (internal/child_process.js:389:16)
    at process._tickCallback (internal/process/next_tick.js:152:19)
```

windows 10
node: v9.4.0
yarn: v1.5.1
npm: v5.6.0
webdash: v1.3.4
webdash-package-json: v1.3.1

**Comment by jadjoubran** [comment]
Thanks @alexisab 
When did this problem occur?
* directly when you ran `npx webdash serve`?
* or when you tried to update a package from webdash-npm-scripts plugin?

**Comment by alexisab** [comment]
It occurs when I click on a script's button on webdash-npm-scripts module, on my `start` script here but it's the same for all my scripts.

**Comment by jadjoubran** [comment]
It seems windows related and can be fixed like as mentioned [here](https://stackoverflow.com/a/36562121/1599122)
Would you be up to send a PR? 😄

**Comment by jadjoubran** [comment]
actually please give it a try first, by simply editing this file: `node_modules/webdash-npm-scripts/api.js` and then apply the fix there so we can see if it works (I'm on mac)

**Comment by alexisab** [comment]
Sure ! I will send a PR asap 😃 !

**Comment by jadjoubran** [comment]
Fixed via #13
